feature/experimental/p4rt/ate_tests/google_discovery_protocol_packetin_test/google_discovery_protocol_packetin_test.go

```diff
@@ -78,6 +82,175 @@ var (
 	}
 )
 
+type PacketIO interface {
+	GetTableEntry(delete bool) []*wbb.ACLWbbIngressTableEntryInfo
+	GetPacketTemplate() *PacketIOPacket
+	GetTrafficFlow(ate *ondatra.ATEDevice, frameSize uint32, frameRate uint64) []*ondatra.Flow
+	GetEgressPort() []string
+	GetIngressPort() string
+}
+
+type PacketIOPacket struct {
+	SrcMAC, DstMAC *string
+	EthernetType   *uint32
+}
+
+type testArgs struct {
+	ctx      context.Context
+	leader   *p4rt_client.P4RTClient
+	follower *p4rt_client.P4RTClient
+	dut      *ondatra.DUTDevice
+	ate      *ondatra.ATEDevice
+	top      *ondatra.ATETopology
+	packetIO PacketIO
+}
+
+// programmTableEntry programs or deletes p4rt table entry based on delete flag.
+func programmTableEntry(ctx context.Context, t *testing.T, client *p4rt_client.P4RTClient, packetIO PacketIO, delete bool) error {
+	t.Helper()
+	err := client.Write(&p4_v1.WriteRequest{
+		DeviceId:   deviceId,
+		ElectionId: &p4_v1.Uint128{High: uint64(0), Low: electionId},
```

Comment:
Similarly please make it electionID

Reply:
sure. I made similar changes to deviceId, electionId and portId.